app/views/_mobile-service-instance-row.html

```diff
@@ -0,0 +1,81 @@
+          <span 
+            class="logo {{row.serviceClass.spec.externalMetadata['console.openshift.io/iconClass']}}" 
+            ng-if="row.serviceClass.spec.externalMetadata['console.openshift.io/iconClass'] && !row.serviceClass.spec.externalMetadata.imageUrl" ></span>
+          <a ng-href="{{row.apiObject | navigateResourceURL}}" ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></a>
+          <div ng-bind-html="row.apiObject.metadata.name | highlightKeywords : row.state.filterKeywords" class="list-row-longname"></div>
+        </h3>
+        <button class="btn btn-default" type="button" ng-if="!row.binding && !row.expanded" ng-click="row.showOverlayPanel('bindService', {target: row.apiObject})">Create Binding</button>
+      </div>
+    </div>
+    <div class="list-pf-actions">
+      <div class="dropdown-kebab-pf" uib-dropdown ng-if="row.actionsDropdownVisible()">
+        <button uib-dropdown-toggle class="btn btn-link dropdown-toggle">
+          <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
+          <span class="sr-only">Actions</span>
+        </button>
+        <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu role="menu">
+          <li role="menuitem" ng-if="!row.binding && (row.serviceBindingsVersion | canI : 'create')">
+            <a href="" ng-click="row.showOverlayPanel('bindService', {target: row.apiObject})">
+              Create Binding
+            </a>
+          </li>
+          <li role="menuitem" ng-if="row.binding && (row.serviceBindingsVersion | canI : 'delete')">
+            <a href="" ng-click="row.showOverlayPanel('unbindService', {target: row.apiObject})">Delete Binding</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+  <div class="list-pf-expansion collapse" ng-if="row.expanded" ng-class="{ in: row.expanded }">
+    <div class="list-pf-container">
+      <div class="expanded-section no-margin">
+        <h4 ng-if="row.binding" class="component-label section-label">Details</h4>
+        <ul ng-if="row.binding">
+            <li ng-if="row.serviceClass.spec.externalMetadata.documentationUrl | size">
+              <b>Documentation</b>:
+              <a href="{{row.serviceClass.spec.externalMetadata.documentationUrl}}">SDK Setup <i class="fa fa-external-link" aria-hidden="true"></i></a>
+            </li>
+            <li ng-repeat="annotation in row.annotations">
+                <b>{{annotation.label}}</b>: 
+                <a ng-if="annotation.type === 'href'" href="{{annotation.value}}">{{ annotation.value }}</a>
+                <span ng-if="annotation.type === 'string'">{{ annotation.value }}</span>
+            </li>
+        </ul>
+        <span class="note" ng-if="row.binding && (row.annotations | size) == 0">No configuration data to show for this service.</span>
+        <div ng-if="!row.binding" class="row create-binding">
+            <div class="col-md-12">
+                <span class="fa fa-info-circle icon"></span>
```

Comment:
Can this icon be changed to the patternfly version (`pficon-info`) with a colour of `#888` to match the mockups?